packages/seal-mongodb-adapter/composer.json

```
@@ -0,0 +1,79 @@
+{
+    "name": "cmsig/seal-mongodb-adapter",
+    "description": "An adapter to support MongoDB in cmsig/seal search abstraction.",
+    "type": "library",
+    "license": "MIT",
+    "keywords": [
+        "cmsig",
+        "seal",
+        "seal-adapter",
+        "search-client",
+        "mongodb"
+    ],
+    "autoload": {
+        "psr-4": {
+            "CmsIg\\Seal\\Adapter\\MongoDB\\": "src"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "CmsIg\\Seal\\Adapter\\MongoDB\\Tests\\": "tests"
+        }
+    },
+    "authors": [
+        {
+            "name": "Alexander Schranz",
+            "email": "alexander@sulu.io"
+        }
+    ],
+    "require": {
+        "php": "^8.1",
+        "cmsig/seal": "^0.7",
+        "mongodb/mongodb": "^1.20"
```

Please use the latest versions, inlcuding 2.0:
```suggestion
        "mongodb/mongodb": "^1.21 || ^2.0"
```